### PR TITLE
Split script

### PR DESCRIPTION
**Pull request by prerit2010**

### CODE REVIEW

**Comment by rgaiacs** [comment]
Looks good.

File: api.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    except NameError:
+        pass
+    choice = input("\nPlease share your email id and workshop_id with us " \
+                    "to improve our scripts.\nAre you in ? (y/n) : ")
+    if choice == 'y' or choice == 'Y':
+        email = input("Please Enter your email id: ")
+        data['user_system_info']['email_id'] = email
+        workshop_id = input("Please Enter the workshop id: ")
+        data['user_system_info']['workshop_id'] = workshop_id
+    senddata()
```
Comment (pbanaszkiewicz) [inline api.py]: Your editor doesn't add empty lines at the end of files.

File: swc-main.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    return True
+
+if __name__ == '__main__':
+
+    if python_version_check():
+        print('Passed')
+    else:
+        print('Failed')
+        print('Install a current version of Python!')
+        print('https://www.python.org/downloads/release/python-352/')
```
Comment (pbanaszkiewicz) [inline swc-main.py]: I'd change that to `Install the newest version from https://www.python.org/downloads/ or ask instructors for help`.

**Comment by pbanaszkiewicz** [comment]
This is looking good :-)

**Comment by pbanaszkiewicz** [comment]
@prerit2010 will you update & merge this to master?

**Comment by pbanaszkiewicz** [comment]
I'm pretty sure you need to update `README` as well.

**Pull request merged**